Library/Formula/sdl_ttf.rb

```diff
@@ -18,8 +18,13 @@ class SdlTtf < Formula
   depends_on "sdl"
   depends_on "freetype"
 
+  # Fix broken TTF_RenderGlyph_Shaded()
+  # https://bugzilla.libsdl.org/show_bug.cgi?id=1433
+  patch :DATA
+
   def install
     ENV.universal_binary if build.universal?
+    inreplace "SDL_ttf.pc.in", "@prefix@", HOMEBREW_PREFIX
```

Comment:
Oh, ugh. I forgot about SDL's prefix fun. We may well be stuck with `HOMEBREW_PREFIX` then if we absolutely have to inreplace the path. It is indeed the same as we apply for `SDL2`.

> And it wouldn't be easy to patch only this specific location with inreplace.

Alright, fair enough. It might be nicer to `gist` it or something and then apply it with `patch do`. Inline patches tend to get sloppy where people accidentally add or remove whitespacing from them on other changes and don't notice.